src/index.tsx

```diff
@@ -228,7 +228,10 @@ export class App extends React.Component<any, AppState> {
 
   public shouldComponentUpdate() {
```

Comment:
```js
shouldComponentUpdate(nextProps, nextState)
```

Can you try

```js
this.state = nextState;
this.componentUpdate();
```

and see if that works?

Otherwise i'll need to figure out another way.

Reply:
Seems to work. But still not a long-term fix. The best fix is what I proposed in #592 -- split the `<App/>` compo. But let's not do it now.